tests/cloud_playbooks/create-gce.yml

```diff
@@ -5,7 +5,7 @@
   vars:
     cloud_machine_type: g1-small
     mode: default
-
+    preemptible: no
```

Comment:
can you add it as gitlab variables too and do:
```-e $GCE_PREEMPTIBLE```  in the ansible options ?

Reply:
I can, but let's default to no, then in a follow up pr try enabling it.

Reply:
Yes, default to no